src/changelog.ts

```diff
@@ -93,18 +117,49 @@ function stringifyReleases(
   return [stringifiedUnreleased, ...stringifiedReleases].join('\n\n');
 }
 
+/**
+ * Return the given URL with a trailing slash. It is returned unaltered if it
+ * already has a trailing slash.
+ *
+ * @param url - The URL string.
+ * @returns The URL string with a trailing slash.
+ */
 function withTrailingSlash(url: string) {
   return url.endsWith('/') ? url : `${url}/`;
 }
 
+/**
+ * Get the GitHub URL for comparing two git commits.
+ *
+ * @param repoUrl - The URL for the GitHub repository.
+ * @param firstRef - A reference (e.g. commit hash, tag, etc.) to the first commit to compare.
+ * @param secondRef - A reference (e.g. commit hash, tag, etc.) to the second commit to compare.
```

Comment:
Nit:
```suggestion
 * @param firstRef - A reference (e.g., commit hash, tag, etc.) to the first commit to compare.
 * @param secondRef - A reference (e.g., commit hash, tag, etc.) to the second commit to compare.
```

Reply:
TIL that this is the proper way to format it in US English. In Canada we don't use the comma.